recipes/nsync/all/conanfile.py

```diff
@@ -0,0 +1,52 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class NsyncConan(ConanFile):
+    name = "nsync"
+    description = "Library that exports various synchronization primitive"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("c", "thread", "multithreading", "google")
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {"with_tests": [True, False]}
```

Comment:
does library support shared and static configurations?
"shared" and "fPIC" options should be added